Code/Framework/AzFramework/Tests/QualitySystemComponentTests.cpp

```diff
@@ -91,15 +94,24 @@ namespace UnitTest
         ASSERT_TRUE(result);
 
         // when the quality system component registers group cvars
-        m_qualitySystemComponent->Activate();
+        AZ::ComponentApplication::Descriptor desc;
+        AZ::ComponentApplication::StartupParameters startupParameters;
+        startupParameters.m_loadSettingsRegistry = false;
+        startupParameters.m_loadAssetCatalog = false;
+        m_application->Start(desc, startupParameters);
+
+        // Without this, the user settings component would attempt to save on finalize/shutdown. Since the file is
+        // shared across the whole engine, if multiple tests are run in parallel, the saving could cause a crash
+        // in the unit tests.
+        AZ::UserSettingsComponentRequestBus::Broadcast(&AZ::UserSettingsComponentRequests::DisableSaveOnFinalize);
```

Comment:
Why not do this logic to Start the AzFramework Application inside of the Fixture SetUp function?

That way every test wouldn't have to add this code.

Reply:
I am waiting to start the application so I can populate the settings registry with the test JSON values before the `QualitySystemComponent` is activated as part of application start up, otherwise the quality settings won't be in the registry and won't get created, but you're right that I can at least make this common code a fixture function that accepts the JSON to test.